api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -146,6 +147,9 @@ public void started() {
             healthCenter.setName(administrationService.getGlobalProperty(AijarConstants.GP_HEALTH_CENTER_NAME));
             locationService.saveLocation(healthCenter);
 
+            EnableDisablePatientFlags enableDisablePatientFlags = new EnableDisablePatientFlags();
```

Comment:
@slubwama No need for a separate class a single function in the activator should work fine since its a single SQL statement